Source/appfat.cpp

```diff
@@ -289,7 +289,7 @@ BOOL InsertCDDlg()
 	return nResult == 1;
 }
 
-void DirErrorDlg(char *error)
+void DirErrorDlg(const char *error)

```

Comment:
This is likely an issue upstream, if you could investigate that and then possibly upstream it that would be a big help.

Reply:
Basically all the dialog functions should accept `const char *` instead of `char *`. I'll send a PR but will need someone else to check bin exactness.

Reply:
Sent https://github.com/diasurgical/devilution/pull/1749